L'exécution de votre programme a déclenché une exception java.lang.ArrayIndexOutOfBoundsException : 4
 Merci de corriger votre code.
   at plm.runtime20.LangtonsAnt.trouver$1 (plm.runtime20.LangtonsAnt:16)
   at plm.runtime20.LangtonsAnt.utiliser$1 (plm.runtime20.LangtonsAnt:27)
   at plm.runtime20.LangtonsAnt.step (plm.runtime20.LangtonsAnt:44)
   at plm.runtime20.LangtonsAnt$$anonfun$run$2.apply$mcVI$sp (plm.runtime20.LangtonsAnt:58)
   at scala.collection.immutable.Range.foreach$mVc$sp (Range.scala:141)
   at plm.runtime20.LangtonsAnt.run (plm.runtime20.LangtonsAnt:56)
   at plm.core.lang.JVMCompiledLang.runEntity (null:-1)
   at plm.universe.World$1.run (null:-1)
   at java.lang.Thread.run (null:-1)
